Makefile.am

```diff
@@ -43,9 +43,7 @@ AM_CFLAGS = @STRICT_CFLAGS@
 # Make sure to run Gtk-doc tests and build the documentation when doing
 # 'make distcheck'
 AM_DISTCHECK_CONFIGURE_FLAGS = \
-	--enable-gtk-doc \
-	--enable-gir-doc \
-	--enable-strict-flags \
+	--enable-gtk-doc
```

Comment:
Sorry, that was a typo. I meant to say that you could do `AM_DISTCHECK_CONFIGURE_FLAGS = --enable-gtk-doc` if you wanted to put everything on one line. We don't want `AM_DISTCHECK_CONFIGURE_FLAGS` to be empty.